frontend/javascripts/oxalis/view/settings/export_bounding_box_modal.js

```diff
@@ -68,6 +88,10 @@ const ExportBoundingBoxModal = ({ destroy, dataset, boundingBox, tracing }: Prop
         annotationType: null,
         tracingVersion: null,
         hasMag1: hasMag1(layer),
+        hideUnmappedIds: existsActivePersistentMapping ? hideUnmappedIds : null,
+        mappingName: existsActivePersistentMapping ? mappingName : null,
+        mappingType: existsActivePersistentMapping ? mappingType : null,
+        isColorLayer: layer.category === "color",
```

Comment:
What happens when this code runs for a color layer while a persistent mapping is active? This will pass the mapping parameters to the back-end which is confusing, I'd assume? Maybe

```suggestion
        // ... somewhere above this block
        const isColorLayer = layer.category === "color";
        // ...
        hideUnmappedIds: !isColorLayer && existsActivePersistentMapping ? hideUnmappedIds : null,
        mappingName: !isColorLayer && existsActivePersistentMapping ? mappingName : null,
        mappingType: !isColorLayer && existsActivePersistentMapping ? mappingType : null,
        isColorLayer
```